src/event.njk

```diff
@@ -0,0 +1,27 @@
+---
+layout: layouts/layout.njk
+tags:
+  - eventPages
+pagination:
+  alias: event
+  data: events
+  size: 1
+  addAllPagesToCollections: true
+renderData:
```

Comment:
I _think_ the fix would be to use `eleventyComputed` then you won't need the `or` check in the template, it will pick it up correctly as just `title`